plasmapy/diagnostics/langmuir.py

```diff
@@ -108,22 +111,23 @@ def obtain_EEDF(potential_sweep, measured_current, probe):
     """
     return
```

Comment:
Coveralls complains that none of the returns get tested. Shouldn't be a problem once there is actually a value to return and a simple test for the function, but just a heads up. Same goes for all the other empty returns.